### PR TITLE
Version bump, changelog, notices for 1.1.0

### PR DESCRIPTION
**Pull request by bwateratmsft**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+## 1.1.0 - 21 Apr 2020
+### Added
+* Custom file names for docker-compose files can be defined [#102](https://github.com/microsoft/vscode-docker/issues/102)
+* The experience for pushing Docker images has been revamped [#351](https://github.com/microsoft/vscode-docker/issues/351), [#1539](https://github.com/microsoft/vscode-docker/issues/1539), [#1595](https://github.com/microsoft/vscode-docker/issues/1595)
+* Extensibility model for registry providers has been improved [#147](https://github.com/microsoft/vscode-docker/issues/147)
+* Generic DockerV2 registries using OAuth can now be connected to in many cases [#869](https://github.com/microsoft/vscode-docker/issues/869)
+
```
Comment (karolz-ms) [inline CHANGELOG.md:7]: What about the new feature that we now react to Docker context changes? That includes context commands in the palette.

Comment (bwateratmsft) [reply CHANGELOG.md:7]: Good idea, I'll add that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
+## 1.1.0 - 21 Apr 2020
+### Added
```
Comment (ravipal) [inline CHANGELOG.md:2]: 'Add Docker Compose...' ?

Comment (ravipal) [inline CHANGELOG.md:2]: Docker installer support?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+* Custom file names for docker-compose files can be defined. [#102](https://github.com/microsoft/vscode-docker/issues/102)
+* The experience for pushing Docker images has been revamped. [#351](https://github.com/microsoft/vscode-docker/issues/351), [#1539](https://github.com/microsoft/vscode-docker/issues/1539), [#1595](https://github.com/microsoft/vscode-docker/issues/1595)
+* Extensibility model for registry providers has been improved. [#147](https://github.com/microsoft/vscode-docker/issues/147)
+* Generic DockerV2 registries using OAuth can now be connected to in many cases. [#869](https://github.com/microsoft/vscode-docker/issues/869)
+* Docker contexts can now be changed, inspected, and removed from the Command Palette. [#1784](https://github.com/microsoft/vscode-docker/issues/1784)
+* If the Docker context is changed from outside VSCode, the changes will be picked up in VSCode within 20 seconds by default, configurable with the `docker.contextRefreshInterval` setting. If the Docker context is changed within VSCode it is picked up immediately. [#1790](https://github.com/microsoft/vscode-docker/pull/1790)
+
+### Fixed
```
Comment (ravipal) [inline CHANGELOG.md:10]: Isn't there more fixes worth mentioning here? How about #1583

File: CHANGELOG.md (hunk 2 above)
Comment (bwateratmsft) [reply CHANGELOG.md:2]: Very few people will ever encounter this, and it's not a feature users will get excited about.

File: CHANGELOG.md (hunk 3 above)
Comment (bwateratmsft) [reply CHANGELOG.md:10]: Added 1583

File: CHANGELOG.md (hunk 2 above)
Comment (bwateratmsft) [reply CHANGELOG.md:2]: We've had the ability to add compose files for a very long time; that it is now split into another command isn't particularly interesting to users.

**Pull request merged**